macros/netatalk.m4

```diff
@@ -52,6 +52,13 @@ AC_DEFUN([AC_NETATALK_DTRACE], [
     [WDTRACE=$withval],
     [WDTRACE=auto]
   )
+  case "$host_os" in
+  *darwin*)
+  if test x"$WDTRACE" = x"auto" ; then
+  WDTRACE=no
+  fi
+  ;;
+  esac
```

Comment:
Might be worth leaving a comment here to explain why dtrace is being disabled.

Reply:
As above!

Reply:
**Update:** done